python/rmmbr/main_test.py

```diff
@@ -7,6 +7,14 @@
 from rmmbr import cloud_cache, local_cache, mem_cache
 
 
+async def wait_all_tasks():
+    all_tasks = asyncio.all_tasks()
+    if current_task := asyncio.current_task():
```

Comment:
iiuc we always enter this `if` (?)

Reply:
current_task can return None, maybe we can `never` this our or just ignore the typecheck fail, but I prefer not to

Reply:
ah if it's for typing just `assert`
having an `if` with a constant results misleads the reader, while `assert` informs the reader